tasks/main.yml

```diff
@@ -1,12 +1,22 @@
 ---
-- name: Install targetcli packages
-  yum:
-    name: 'targetcli'
+
+- include_vars: vars_RedHat_CentOS.yml
+  when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
+
+- include_vars: vars_SUSE_openSUSE.yml
+  when: '(ansible_os_family == "Suse" or ansible_os_family == "openSUSE Leap")'
+
+- include_vars: vars_SUSE_SLES12.yml
+  when: 'ansible_os_family == "Suse" and ansible_distribution_major_version|int < 15'
```

Comment:
As this will apply only to version `12`, could this be just `== 12` ?

Reply:
In theory someone could try to use this with SLES11. SLES11 SP4 is still supported by SUSE.

Not sure if it will work there...

Reply:
My practical question here if you have tried/tested it in SLES 11?
This version list doesn't prevent anyone from using it only on specific versions but it gives at least idea on where it should work. My approach to this is to keep the list specific to versions that someone has tried the role on and it worked as intended.
**[?]** Would you be able to test it on SLES 11 so that can be added to that list or should we just leave it out?